src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -239,9 +239,9 @@
       (str $ address))
     address))
 
-(defn get-standard-fiat-format
-  [crypto-value currency-symbol fiat-value]
-  (if (string/includes? crypto-value "<")
+(defn fiat-formatted-for-ui
```

Comment:
This function was renamed because we don't have any "standard" fiat format) But naming it as "ui"-related makes it clear that result shouldnt be used in calculations.
Also for some reason we were passing here crypto value and checking if it includes "<" sign. We actually don't need this because we have a fiat value. So extra argument removed.